### PR TITLE
Static vs. dynamic websites

### PR DESCRIPTION
**Pull request by gboone**
May be holding this a few days while we sort out editorial plans. It's here when we need it.

### CODE REVIEW

File: _posts/2016-07-11-conversation-about-static-dynamic-websites.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+might run a few dozen queries or a few hundred depending on what that
+content should be.
+
+So the web server makes the page anew, and sends some HTML over the
+internet to your computer. Finally, your browser figures out how to make
+sense of and render all that HTML. This happens every time someone asks
+for that page.
+
+*So to recap, a CMS stores all the content a website has in a database
+and then jams it into a template when​_you_​ ask for a specific page.*
```
Comment (wslack) [inline _posts/2016-07-11-conversation-about-static-dynamic-websites.md]: displays as "whenyou"

File: _posts/2016-07-11-conversation-about-static-dynamic-websites.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+- content management
+- federalist
+excerpt: "Our blog uses Jekyll, a static file generator with a basic templating system, as the backend software. Deploying our blog posts this way has simplified our publishing process."
+description: "Our blog uses Jekyll, a static file generator with a basic templating system, as the backend software. Deploying our blog posts this way has simplified our publishing process."
+image: /assets/blog/new-jekyll-site/header.png
+---
+
+Our blog uses [Jekyll](http://jekyllrb.com/), a static file
+generator with a basic templating system, as the backend software for
+our site. A few months ago we wrote about [our decision to host the site on Federalist]({{ site.baseurl }}/2016/05/18/why-were-moving-18f-gsa-gov-to-federalist/). This post is about what static site generators are and the advantages and disadvantages of using one over a more traditional content management system (CMS).
```
Comment (wslack) [inline _posts/2016-07-11-conversation-about-static-dynamic-websites.md]: Maybe "This post is about static sites generators and"

File: _posts/2016-07-11-conversation-about-static-dynamic-websites.md (hunk 1 above)
Comment (gboone) [inline _posts/2016-07-11-conversation-about-static-dynamic-websites.md]: Thanks, I couldn't find that before.

**Comment by wslack** [comment]
all good for me @gboone

**Pull request merged**